library/ssl_misc.h

```diff
@@ -2055,20 +2055,58 @@ static inline int mbedtls_ssl_sig_alg_is_supported(
           MBEDTLS_ECDSA_C */
 
 #if defined(MBEDTLS_SHA256_C) && \
-    defined(MBEDTLS_X509_RSASSA_PSS_SUPPORT)
+    defined(MBEDTLS_PKCS1_V21) && \
```

Comment:
But I am confuse on it. Why it is right here ? And in certificate_verify , it is wrong

Reply:
Because we use 'mbedtls_ssl_sig_alg_is_supported' to determine which signature algorithm we propose in the signature_algorithms extension and as we do not send a signature_algorithms_cert extension the algorithms in the signature_algorithms extension can be used as signature algorithms in certificates. And if a certificate is sign with RSA PSS my understanding is that we need MBEDTLS_X509_RSASSA_PSS_SUPPORT to be able to verify it. Does it make sense ?

Reply:
thanks